tests/tests.rs

```diff
@@ -1607,6 +1607,32 @@ fn regression_391() {
     assert_eq!(lines, "bar.py\n");
 }
 
+// See: https://github.com/BurntSushi/ripgrep/issues/451
+#[test]
+fn regression_451_only_matching() {
```

Comment:
This test looks good. Could you add another test that approximately mimics the bug report in #451? e.g., Test that the actual results printed are correct instead of just column numbers.